app/models/user.rb

```diff
@@ -216,4 +201,11 @@ def self.active
     where(deactivated_at: nil)
   end
 
+  def create_default_price_group!
+    return unless SettingsHelper.feature_on?(:user_based_price_groups)
+
+    price_group = email_user? ? PriceGroup.external : PriceGroup.base
+    price_group_members.create!(price_group: price_group)
```

Comment:
Do we have any validations on uniqueness for price group membership? Not sure if it's worth doing. What would happen if we called this method multiple times? Would we end up with multiple memberships? If we prevent that, we can probably avoid the `uniq` scope above.